curriculum/challenges/english/10-coding-interview-prep/data-structures/check-if-binary-search-tree.md

```diff
@@ -39,6 +39,28 @@ assert(
 );
 ```
 
+Your Binary Search Tree should return false when checked with `isBinarySearchTree()`.
```

Comment:
More appropriate could be that `isBinarySearchTree()` will return false when tree isn't a proper binary tree.

Reply:
I amended the commit - I made the requested changes.